src/app/api/v1/slack/rotation-post/route.ts

```diff
@@ -73,6 +73,7 @@ export async function POST(req: NextRequest) {
     );
   } catch (error) {
     console.error('Error processing request:', error);
-    return NextResponse.json({ error: 'Failed to process the request' }, { status: 500 });
+    return NextResponse.json(
+      getSlackMessage(SlackResponseType.Ephemeral, 'error: Processing request'));
```

Comment:
```suggestion
      getSlackMessage(SlackResponseType.Ephemeral, 'error: Processing request' + JSON.stringify(error, null, 2));```
we need the message here